Explorer/Assets/Scripts/ECS/Unity/PrimitiveRenderer/Systems/InstantiatePrimitiveRenderingSystem.cs

```diff
@@ -33,16 +33,16 @@ public partial class InstantiatePrimitiveRenderingSystem : BaseUnityLoopSystem
         };
         private readonly IComponentPool<MeshRenderer> rendererPoolRegistry;
         private readonly IComponentPoolsRegistry poolRegistry;
-        private readonly IConcurrentBudgetProvider instantiationFrameTimeBudgetProvider;
+        private readonly IReleasablePerformanceBudget instantiationFrameTimeBudget;
```

Comment:
can be just `IPerformanceBudget` (it doesn't use `Release` method)

Reply:
Good mention, I finished it in last commit